robottelo/helpers.py

```diff
@@ -716,3 +716,64 @@ def idgen(val):
     test name
     """
     return '_parameter'
+
+
+class InstallerCommand:
+    """This class constructs, parses, updates and gets formatted installer commands"""
+
+    def __init__(self, *args, command='satellite-installer', allow_dupes=False, **kwargs):
```

Comment:
allow_dupes - is allow duplicates I suppose :), why not use full naming ?

Reply:
Variable names should be long enough to infer meaning, but not overly verbose.